fix: compliance docs update (#1158)
ory/docs: docs/kratos/concepts/security.mdx

title: Threat models and security profiles
---

Threat modeling is essential for securing a system like Ory Identities. This document provides guidelines for identifying and

> analyzing threats and implementing security measures for Ory Identities and self-hosted Ory Kratos Identity Server.

Author: I think in this instance it is OK 😅

> Running software that stores personal information carries risk. It is important to identify threats and understand the system's
> attack surface, the likelihood, and the impact of an attack.

Author: ```suggestion
Working with and administering software that stores personal information carries risk. It is important to identify threats and understand the system's
attack surface, the likelihood, and the impact of an attack.
```

?

In the case of Ory Identities, threats could include hackers or malicious insiders who may try to steal or manipulate personal
information. To protect against these threats, it is important to properly configure Ory and implement appropriate security
measures, such as strong authentication and access controls. Regular monitoring and timely response to security incidents are also
crucial.

## Digital identity guidelines

There is no universally accepted standard for digital identity.  
Ory follows [Digital Identity Guidelines](https://pages.nist.gov/800-63-3/) established by the
[National Institute of Standards and Technology (NIST)](https://www.nist.gov/).  
These guidelines are accompanied by a [FAQ](https://pages.nist.gov/800-63-3/) that provides additional information and answers to
common questions.

> ## Brute-force defenses

Author: It is defense **against** brute-force. 

If I write **anti-brute force defense**, I am defending against anti-brute force attacks (?)

> ## Brute-force defenses

Reviewer: Yes, that's what it does. But looking at this after a few days it doesn't really sound right. The best solution would be to use a heading that is constructed a bit differently than the rest: 

`Defenses against brute-force attacks`

> ## Brute-force defenses

Author: done!

> Ory Network provides Ory Identities with protection against brute force attacks by rate-limiting requests to API public endpoints,

Reviewer: ```suggestion
Ory Network provides Ory Identities with protection against brute-force attacks by rate limiting requests to API public endpoints,
```
Adjective b4 noun = hyphen
rate limiting = no hyphen (I know Grammarly tells you to use a hyphen, but "rate limiting" here is not describing a quality of "requests")

for example login and registration endpoints.

When self-hosting the Ory Kratos Identity Server, it is the responsibility of the administrator to implement and manage

> rate-limiting or other measures to ensure the security of the network.

Reviewer: Mixing Ory Network and self-hosted instances

## Account takeover defenses

The [settings flow](../self-service/flows/user-settings) implements account takeover defenses by requiring the current password
before allowing the user to change it. This makes it impossible for an attacker to change the password without knowing the

> existing password.

Reviewer: Technically changing passwords is possible only when users have a privileged session. To get a privileged session, users must authenticate. Do you think that it's a good idea to describe this?

> ## Password Policy

Reviewer: ```suggestion
## Password policy
```

> To learn more about setting up a secure password policy, please refer to the

Reviewer: ```suggestion
To learn more about setting up a secure password policy, refer to the
```

No need to be overly polite
